### PR TITLE
Fix horizontal scroll bug on CS page. New Quote styling. Add headers to callout content.

### PR DESCRIPTION
**Pull request by sirleech**

### CODE REVIEW

File: src/cards/cards_with_intro.js [hunk 1]
```diff
@@ -12,8 +12,8 @@ const CardsWithIntro = ( page ) => (
 	<div className="au-body au-grid">
 		<div className="container-fluid">
 			<div className="row">
-				<div className={"col-lg-offset-3 col-md-offset-3 col-md-9 col-lg-10 cards cardswithintro__container "+ (page.oddrow ? 'homepagebanner__oddrow' : '')}>
-					<div className="col-lg-3 cardswithintro__intro">
+				<div className={"col-lg-offset-3 col-md-offset-3 col-md-9 col-lg-9 cards cardswithintro__container "+ (page.oddrow ? 'homepagebanner__oddrow' : '')}>
+					<div className="col-lg-4 cardswithintro__intro">
```
Comment (shad0wMe0w) [inline src/cards/cards_with_intro.js]: 👍

File: src/sass/components/cards.scss [hunk 2]
```diff
@@ -163,7 +162,7 @@
 }
 
 .cards .cards__list > .cards__list__item {
-	margin-bottom: 1em;
+	margin-bottom: 1.4rem;
```
Comment (shad0wMe0w) [inline src/sass/components/cards.scss]: @sirleech quick question, how often should we use the `@include AU-space(css-property, 1unit);` functions for single properties?

Comment (shad0wMe0w) [reply src/sass/components/cards.scss]: As in, does this need to be 

`@include AU-space(margin-bottom, 1.4unit);`?

Comment (sirleech) [reply src/sass/components/cards.scss]: We should from now on (parts of the code don't use it.) I can fix this , good catch

Comment (sirleech) [reply src/sass/components/cards.scss]: Bah-- the rest of the code needs to be refactored. Might leave it for now. New CSS files for sure use the @include AU-space as it gives old browser fallbacks

File: src/sass/base/vd-tweaks.scss [hunk 3]
```diff
@@ -84,3 +84,9 @@ a:focus h1 {
 	border: none;
 	font-weight: 300;
 }
+
+.spacer__med {
+	@include AU-media( md ) {
+		margin-bottom: 3rem;
```
Comment (shad0wMe0w) [inline src/sass/base/vd-tweaks.scss]: @sirleech and this bit?

`@include AU-space(margin-bottom, 3unit);`

File: src/sass/components/cards.scss (hunk 2 above)
Comment (shad0wMe0w) [reply src/sass/components/cards.scss]: Yeah fair enough. I was wondering where we'd start.

**Comment by shad0wMe0w** [comment]
Just waiting for circleCI to finish...

**Comment by shad0wMe0w** [comment]
Done

File: src/sass/base/vd-tweaks.scss (hunk 3 above)
Comment (sirleech) [reply src/sass/base/vd-tweaks.scss]: Here is the margin fix:
https://guides.service.gov.au/content-strategy/identify-business-needs/

Added margin bottom:
https://service-manual.apps.y.cld.gov.au/content-strategy/identify-business-needs/

**Pull request merged**